### ISSUE TITLE
CoP 1-3 - The Mothercrystals - Can black screen when teleporting if Stray's corpses are still around

### ISSUE THREAD

**Issue by Nireya** [GoldSaucerServer/bugtracker#98]
**_I have:_**

- [x] searched existing issues (https://github.com/GoldSaucerServer/bugtracker/issues) to see if the issue has already been opened.
- [x] downloaded and am using Gold Saucer's custom .dat files.
- [x] provided my client version in the space provided below (type /ver into your in-game chat).

**Version**: 30200731_2

**_Additional Information_** (Steps to reproduce/Expected behavior) **:**

Basically the teleporters worked perfectly when we did Promies except there was one instance where someone black screened when we teleported and had to force quit/relog since the Stray's corpse had not disappeared yet.

**Comment by tankfest** [comment]
Issue forwarded:

https://github.com/project-topaz/topaz/issues/1023
